### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Goralive**

### CODE REVIEW

**Comment by Goralive** [comment]
Dunno why but I have always this changes on line 1416 kottan but I have never done them

File: README.md [hunk 1]
```diff
@@ -1413,4 +1413,8 @@ I'm [Yury](https://github.com/yuryzaytsev). This is a street cat named Bob. He i
 My name is [Dima](https://github.com/youhymuk) and I'm glad to study here with you.
 I wish you all success!
 
-![kottan](gif/cat-headshot.gif)
\ No newline at end of file
+![kottan](gif/cat-headshot.gif)
+
+### Greetings kottans
```
Comment (nazmariam) [inline README.md:1418]: Hi! Good job!
But there was no need to close previous PRs

**Review by nazmariam — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Goralive) [reply README.md:1418]: I was try to delete 1416 line with different cases :-) Sorry for deleting previous PRs. This won't help) Now I know :-)

**Comment by nazmariam** [comment]
> Dunno why but I have always this changes on line 1416 kottan but I have never done them

This is OK:) Previous student didn't add empty line at the end of the file, thats why every new contributor "make" changes at that line, usually it's better to set adding new empty line automatically in IDE or code editor

**Comment by nazmariam** [comment]
@Goralive look it can be good practice for you if you'll setup your editor and make new commit with empty line to simplify life for next contributor :) what do you think?

**Comment by Goralive** [comment]
Now I see where the issue. Will do

**Review by miroslav-kolomiets — APPROVED** [review]
Well done!

**Pull request merged**